packages/rancher-alerting-drivers/charts/templates/_helpers.tpl

```diff
@@ -1,3 +1,38 @@
+{{- define "system_default_registry" -}}
+{{- if .Values.global.cattle.systemDefaultRegistry -}}
+{{- printf "%s/" .Values.global.cattle.systemDefaultRegistry -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Windows cluster will add default taint for linux nodes,
+add below linux tolerations to workloads could be scheduled to those linux nodes
+*/}}
+
+{{- define "linux-node-tolerations" -}}
+- key: "cattle.io/os"
+  value: "linux"
+  effect: "NoSchedule"
+  operator: "Equal"
+{{- end -}}
```

Comment:
We should be fine because pods will still be assigned to nodes if the target taint does not exist. 
Those charts are curated for Rancher's clusters, so we don't have to put them in values.yaml for now. If we decide to move then we need to do it for all charts :)